### DISCUSSION TITLE
Text highlighting

### DISCUSSION THREAD

**Discussion by anhthu253** [neuefische/questions#297] [Web]
Hi i m working on highlighting text and found range and selection for that by googling. Everytime when one highlights a text, a range object will be created. Then I packed a html element around this object in oder to give it a style or remove the style (e.g color the selection with yellow or removing highlight). But now i m back writing document.creatElement, el. style.classList.add  and so on again. It doesnt look nice I guess. Would appreciate any other idea!

**Comment by marcbendixen** [comment]
Hey @anhthu253  👋 
I'm not quite sure if that's what you mean, but you can take look at the pseudo element `::selection` ([here on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/::selection)). Maybe this is what you need!?

**Reply by anhthu253** [reply]
Hi thank you for your answer but that was not what i meant. I want to high light texts and it should be seen even after the next mouse click events
